Makefile

```diff
@@ -1212,47 +1212,9 @@ driver/compdynlink.mli: $(DYNLINK_DIR)/dynlink.mli \
     driver/compify_dynlink.sh
 	driver/compify_dynlink.sh $< $@
 
-driver/compdynlink_types.cmi: driver/compdynlink_types.mli
-	$(CAMLC) $(COMPFLAGS) -c $<
-
-driver/compdynlink_types.cmo: driver/compdynlink_types.ml \
-    driver/compdynlink_types.cmi
-	$(CAMLC) $(COMPFLAGS) -c $<
-
-driver/compdynlink_types.cmx: driver/compdynlink_types.ml \
-    driver/compdynlink_types.cmi
-	$(CAMLOPT) $(COMPFLAGS) -c $<
-
 # See comment in otherlibs/dynlink/Makefile about these two rules.
```

Comment:
These rules are subsumed by the general compilation rules and the computed dependencies in `.depend`.